blocks/article-body-block/chains/article-body/_children/heading.jsx

```diff
@@ -1,20 +1,13 @@
 import React from "react";
-import styled from "styled-components";
 
-const Heading = ({ element, primaryColor }) => {
+const Heading = ({ element, classPrefix = "" }) => {
```

Comment:
If it's optional, that means that `__h1` is valid... no?

Reply:
Updated